docs/releases/4.0.txt

```diff
@@ -28,6 +28,47 @@ The Django 3.2.x series is the last to support Python 3.6 and 3.7.
 What's new in Django 4.0
 ========================
 
+``zoneinfo`` default timezone implementation
+--------------------------------------------
+
+The Python standard library's :mod:`zoneinfo` is now the default timezone
+implementation in Django.
+
+This is the next step in the migration from using ``pytz`` to using
+:mod:`zoneinfo`. Django 3.2 allowed the use of non-``pytz`` time zones. Django
+4.0 makes ``zoneinfo`` the default implementation. Support for ``pytz`` is now
+deprecated and will be removed in Django 5.0.
+
+:mod:`zoneinfo` is part of the Python standard library from Python 3.9. The
+``backports.zoneinfo`` package is automatically installed alongside Django if
+you are using Python 3.8.
+
+To the extent that you are always operating on aware datetimes with UTC, then
```

Comment:
I think that's exceedingly conservative. Conversions in forms and templates are also transparent. The only thing that isn't transparent is explicit operations on non-UTC timezones in the model layer.

Reply:
I'll rephrase. Thanks.